cocotb/share/lib/fli/FliObjHdl.cpp

```diff
@@ -221,8 +225,13 @@ long FliEnumObjHdl::get_signal_value_long()
     }
 }
 
-int FliEnumObjHdl::set_signal_value(const long value)
+int FliEnumObjHdl::set_signal_value(const long value, const gpi_set_action_t action)
 {
+    if (action != GPI_DEPOSIT) {
+        LOG_CRITICAL("Force or release action not supported for FLI.");
+        return -1;
+    }
```

Comment:
Nice, thanks - with this in place, I don't think we need to wait for FLI support before merging